src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/OrganisationCreationRequestValidator.java

```diff
@@ -134,20 +130,21 @@ public void requestValues(String... values) {
     public void requestContactInformation(List<ContactInformationCreationRequest> contactInformations) {
         if (null != contactInformations) {
 
-            for (ContactInformationCreationRequest contactInformation : contactInformations) {
+            contactInformations.stream()
+                    .forEach(contactInformation -> {
+                        if (isEmptyValue(contactInformation.getAddressLine1()) || isEmptyValue(contactInformation.getAddressLine2())
```

Comment:
Filters should be used instead if statements

Reply:
filter() method which can be used to filter out some elements from object collection based on a particular condition. here i am not collecting any collection  just validating the request object.